source/gui/settingsDialogs.py

```diff
@@ -1640,27 +1642,28 @@ def makeSettings(self, settingsSizer):
 			("speech", self.driver.name, "delayedCharacterDescriptionsTimeoutMs")
 		).kwargs["max"])
 
-		# Translators: This is a label for a setting in voice settings (an edit box to change
-		# Time for the delayed character descriptions in ms;
+		# Translators: This is a label for a setting in voice settings,
+		# an edit box to change the time for the delayed character descriptions in milliseconds.
 		delayedCharacterDescriptionsTimeoutMsLabel = _("&Time for delayed character descriptions (ms)")
 		self.delayedCharacterDescriptionsTimeoutMsEdit = settingsSizerHelper.addLabeledControl(
 			delayedCharacterDescriptionsTimeoutMsLabel,
 			nvdaControls.SelectOnFocusSpinCtrl,
 			min=minDelay,
 			max=maxDelay,
-			initial=config.conf["speech"][self.driver.name]["delayedCharacterDescriptionsTimeoutMs"])
+			initial=config.conf["speech"][self.driver.name]["delayedCharacterDescriptionsTimeoutMs"],
+		)
 		self.bindHelpEvent(
 			"delayedCharacterDescriptionsTimeoutMs",
 			self.delayedCharacterDescriptionsTimeoutMsEdit
 		)
 		if not config.conf["speech"][self.driver.name]["delayedCharacterDescriptions"]:
-			self.delayedCharacterDescriptionsTimeoutMsEdit.Hide()
+			self.delayedCharacterDescriptionsTimeoutMsEdit.Disable()
 
-	def onToggleDelayDescriptions(self, evt):
+	def onToggleDelayDescriptions(self, evt: wx.CommandEvent) -> None:

```

Comment:
This should enable/disable rather than show/hide.
Show/hide causes visual formatting issues and can be confusing. Enable/disable makes it clear what selecting the checkbox does.